index.html

```diff
@@ -9,26 +9,183 @@
+    			<div class="project-meta-data">
+    				<div class="project-client">Gnome</div>
+    				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-tech">Gnome User</div>
+				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-year">2018</div>
+			</div>
+			<ul class="project-tags">
+
+				<li class="tags">html</li>
+				<li class="tags">css</li>
+				<li class="tags">javaScript</li>
+			
+			</ul>
+			<div class="project-description">
+                    		A daily selection of privately personalized reads; no accounts or sign-ups required.
+                	</div>
+			<div class="project-button">
+                		<button type="button">See Project</button>
+            		</div>
+                </div>
+	</div>
+	
+    	<div class="project">
+    		<img src="projects/project3.png" alt="project 3" title="Project 3" width=295" height="220" />
+    		<div class="project-heading">
+    			<h2 class="project-title">Project 3</h2>
+    			<div class="project-meta-data">
+    				<div class="project-client">Linux</div>
+    				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-tech">Linux User</div>
+				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-year">2020</div>
+			</div>
+			<ul class="project-tags">
+
+				<li class="tags">html</li>
+				<li class="tags">css</li>
+				<li class="tags">javaScript</li>
+			
+			</ul>
+			<div class="project-description">
+                    		A daily selection of privately personalized reads; no accounts or sign-ups required.
+                	</div>
+			<div class="project-button">
+                		<button type="button">See Project</button>
+            		</div>
+                </div>
+	</div>      
+    				
+    	<div class="project">
+    		<img src="projects/project4.png" alt="project 4" title="Project 4" width=295" height="220" />
+    		<div class="project-heading">
+    			<h2 class="project-title">Project 4</h2>
+    			<div class="project-meta-data">
+    				<div class="project-client">Gnome</div>
+    				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-tech">Gnome User</div>
+				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-year">2022</div>
+			</div>
+			<ul class="project-tags">
+
+				<li class="tags">html</li>
+				<li class="tags">css</li>
+				<li class="tags">javaScript</li>
+			
+			</ul>
+			<div class="project-description">
+                    		A daily selection of privately personalized reads; no accounts or sign-ups required.
+                	</div>
+			<div class="project-button">
+                		<button>See Project</button>
+            		</div>
+                </div>
+	</div>
+	</div>
+	
+	<div class="about-section">
```

Comment:
- [ ] Kindly note that the container for the whole my about me section is supposed to be a `<section>` and not a `<div>`. `<section>` is a semantically better way to mark up your code.

- For a visual of what I mean:
 ```
<section class="about-section">
 <!-- Your code here -->
</section>
```